### ISSUE TITLE
Build of Zoneminder with Plugin-support fails

### ISSUE THREAD

**Issue by Pinusno** [manupap1/libzoneminder-plugin-openalpr#4]
Hi,

when I build your ZoneMinder Repo (Jessie, Ubuntu 12.04), then it fails with compile errors like:

zmc.cpp: In function ‘int main(int, char**)’:
zmc.cpp:116:5: error: ‘cout’ was not declared in this scope

It looks like there is something not included, but I could not find what it is missing. When I compile the original Zoneminder Repo then it compiles fine. But also the g++ command is equal to the normal ZoneMinder Git and the files too. 

Do you have any ideas?

Many thanks,
Karsten

**Comment by manupap1** [comment]
Hi,

I can't reproduce this issue but looking at the code I see that iostream is not included in the proper place. This should be done in zmc.cpp or in a close header file but definitively not in zm_image_analyser.h.

Also the std namespace declaration can be suspected. You can try to rebuild zm after prefixing `cout` with `std::` at line 116 of zmc.cpp and check if compilation still fails at this line.

Emmanuel

**Comment by Pinusno** [comment]
Hi Emmanuel,

thank you for your fast answer. Yes to resolve this problem it is a simple task, but after that there came other compiler errors and I think to resolve that, then I would have unexpected behaviors But I tried it now on a clean VM and it was functional. I only could not understand, why there are errors on compile time with the same sources like the normal zoneminder git, But maybe there was an failure before, which I did not saw.

Many thanks
Karsten
